user-guide/network/nictag.rst

```diff
@@ -19,10 +23,19 @@ For other use cases, it is possible to add following NIC tags:
 Managing NIC Tags
 =================
 
-The **nictagadm** command is used for working with NIC tags on the compute node. It is possible to add, create or modify NIC tags with it.
+The ``nictagadm`` command is used for working with NIC tags on the compute node. It is possible to add, create or modify NIC tags with it.
+
+.. warning:: NIC tags should only be managed using ``nictagadm``; if you use ``dladm`` to create and/or delete NIC tags, unexpected behavior might occur.
+
+There are 4 types of NIC tags available:
+
+- normal
+- aggregate
+- etherstub
+- overlay
 
-Adding a NIC Tag
-----------------
+Adding normal/aggr NIC Tag
```

Comment:
IMO aggr is of a type `normal`. Using `aggr` may be confusing. But it may be obvious from the global concept (I don't see the whole document so I may be wrong).

Reply:
In the lines below creation of the normal and aggr NIC tag is shown, that was the reason why I included the aggr in the titile. Other titles below also include names of etherstubs and overlays.